sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/NanosOffsetClock.java

```diff
@@ -44,15 +40,5 @@ public Instant now() {
         baseMillis + (TimeUnit.MILLISECONDS.convert(
             System.nanoTime() - nanosAtBaseMillis, TimeUnit.NANOSECONDS)));
   }
-
-  /**
-   * Creates instances of {@link NanosOffsetClock}.
-   */
-  public static class Factory implements DefaultValueFactory<Clock> {
```

Comment:
I needed to replace the way clocks are obtained so multiple pipelines don't interact with each other if they're both using `TestStream`; that requires suppliers, which means we can have a default supplier instead of a DefaultValueFactory.